main.tf

```diff
@@ -6,10 +6,10 @@ resource "random_id" "uniq" {
   byte_length = 4
 }
 
-resource "random_string" "external_id" {
-  count            = var.create ? 1 : 0
-  length           = var.external_id_length
-  override_special = "=,.@:/-"
+resource "lacework_external_id" "aws_iam_external_id" {
+  count      = var.create ? 1 : 0
+  csp        = "aws"
+  account_id = var.lacework_aws_account_id
```

Comment:
same issue as the other PR I just reviewed, use the caller AWS account ID, not the Lacework AWS account id.

Reply:
Not sure that this is available in this module  if I look at the variables,

```
variable "lacework_aws_account_id" {
  type        = string
  default     = "434813966438"
  description = "The Lacework AWS account that the IAM role will grant access"
}
```
Is that not the account id that we wish to use?

Reply:
added `data "aws_caller_identity" "current" {}` and used the id from that